src/main/java/com/nextstep/camp/lotto/domain/vo/ExpressionTokens.java

```diff
@@ -0,0 +1,69 @@
+package com.nextstep.camp.lotto.domain.vo;
+
+import com.nextstep.camp.common.utils.NumberUtils;
+import com.nextstep.camp.lotto.domain.exception.NotOddNumberOfValuesException;
+import com.nextstep.camp.lotto.domain.exception.NotStartWithNumberException;
+import com.nextstep.camp.lotto.domain.exception.TooShortExpressionException;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ExpressionTokens {
+
+    private final LinkedList<ExpressionToken> values;
```

Comment:
변수의 타입은 구현체보다 인터페이스로 선언하면 어떨까요?
```suggestion
    private final List<ExpressionToken> values;
```

Reply:
구현체 클래스를 사용한 이유는
- 외부에서 사용하는 연산(poll())이 List 인터페이스에 존재하지 않기 때문입니다.
- LinkedList가 흔히 사용하는 ArrayList 보다 해당 연산에 성능 상 우위에 있기 떄문입니다.
- ExpressionTokens가 이런 종류의 연산을 사용하는 객체임을 분명히하고 싶기 때문입니다.
List로 선언했을 떄는 장점보다, LinkedList로 선언했을 때의 장점이 더 크다고 생각합니다.
피드백 부탁드립니다!